pkg/koordlet/resmanager/cpu_burst.go

```diff
@@ -274,18 +274,18 @@ func (b *CPUBurst) getNodeStateForBurst(sharePoolThresholdPercent int64,
 	sharePoolCPUCoresUsage := nodeCPUCoresUsage
 	for _, podMeta := range podsMeta {
 		podQOS := apiext.GetPodQoSClass(podMeta.Pod)
-		// exclude LSR pod cpu from cpu share pool
-		if podQOS == apiext.QoSLSR {
+		// exclude LSE/LSR pod cpu from cpu share pool
+		if podQOS == apiext.QoSLSE || podQOS == apiext.QoSLSR {
```

Comment:
Thanks for your contribution! We'd better add a UT to verify the fix and avoid regression.